setup.py

```diff
@@ -28,12 +28,13 @@
         "pre-commit",
     ],
     "docs": [
-        "sphinx==4.3.1",
-        "sphinx_rtd_theme==1.0.0",
-        "sphinx_gallery==0.10.1",
-        "m2r2==0.3.1",
-        "mistune<2.0.0",
-        "Pillow==8.4.0",
+        "sphinx==5.1.1",
+        "sphinx_autodoc_typehints==1.19.2",
+        "sphinx_rtd_theme==0.5.1",
+        "sphinx_gallery==0.11.1",
+        "m2r2==0.3.3",
+        "mistune==0.8.4",
```

Comment:
this one has marched on quite a bit: https://pypi.org/project/mistune/#history

but the "<2.0.0" is equivalent to "==0.8.4.", so ok for this PR

Reply:
-> #446